src/templates/search.jsx

```diff
@@ -15,6 +15,8 @@ import Filters from '../components/Filters';
 import ActiveFilters from '../components/ActiveFilters';
 import fetch from 'isomorphic-fetch';
 
+const path = require('path');
```

Comment:
So this will make a path polyfill be added to the front-end bundle. And I'm really not sure what the path is doing. Is it just for issue template? I don't think it needs to be anything but a simple string.

Reply:
That's true, I am not too sure what I was thinking when I added this. Having it as a simple string does make sense. Thanks!